docs/data/material/discover-more/showcase/showcase.md

```diff
@@ -1,13 +1,13 @@
 # Showcase
 
-<p class="description">The following is a list of some of the public apps using MUI.</p>
+<p class="description">Check out these public apps using Material UI to get inspired for your next project.</p>
```

Comment:
Will we have one showcase per product or one showcase for the whole suite?

```suggestion
<p class="description">Check out these public apps using MUI to get inspired for your next project.</p>
```

Reply:
I'd say one showcase per product (inside each one's documentation). This is considering that _showcase_ is one thing and _case study_ (an article that goes more in-depth on a given user experience with MUI [product] — [illustrative benchmark](https://linear.app/customers/loom)) is another.